sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/FlattenTest.java

```diff
@@ -228,7 +228,7 @@ public void testFlattenPCollectionsEmptyThenParDo() {
   public void testFlattenNoListsNoCoder() {
     // not ValidatesRunner because it should fail at pipeline construction time anyhow.
     thrown.expect(IllegalStateException.class);
-    thrown.expectMessage("cannot provide a Coder for empty");
+    thrown.expectMessage("Unable to return a default Coder");
 
     PCollectionList.<ClassWithoutCoder>empty(p)
```

Comment:
[Side note] we should be able to chose an arbitrary coder here as we won't actually be encoding any elements... I suppose we'd have to handle the case of downstream operations creating something from nothing (is that allowed?)